packages/css/src/components/shopping-cart/shopping-cart.template.ts

```diff
@@ -47,18 +47,23 @@ export function shoppingCartTemplate({ collapsed, hideSummary, removeAll, isOpen
           ? html`
             ${!hideSummary
               ? html`
-                <button
-                  type="button"
-                  class="dso-status"
-                  aria-expanded="${ifDefined(collapsed ? 'false' : 'true')}">
-                  ${!collapsed ? html`
-                    ${iconTemplate({ icon: 'chevron-up' })}
-                  `
-                  : html`
-                    ${iconTemplate({ icon: 'chevron-down' })}
-                  `}
-                  U heeft ${countItems(items)} activiteit${((countItems(items) > 1) ? html`en` : nothing )} gekozen
-                </button>
+                ${!collapsable
```

Comment:
In de `packages/dso-toolkit/components/Componenten/shopping-cart/shopping-cart.njk` doe je het precies andersom. Daar zorgt `collapsable` voor in- en uitklapbaarheid.

Reply:
Klopt, dat was een kronkel. Aangepast.